src/Orleans/Core/ClientBuilder.cs

```diff
@@ -81,6 +83,8 @@ private static void AddBasicServices(IServiceCollection services, ClientConfigur
             services.AddFromExisting<IInternalGrainFactory, GrainFactory>();
             services.AddFromExisting<IGrainReferenceConverter, GrainFactory>();
             services.AddSingleton<ClientProviderRuntime>();
+
+            services.AddLogging();
```

Comment:
we can configure a fancier default logger when we upgrade to Microsoft.Extensions.Logging 2.0 . 
using 
`services.AddLogging(builder => builder.AddConsole())` or similar. currently LogBuilder is not available in 1.3